lib/ansible/modules/cloud/vmware/vmware_guest.py

```diff
@@ -254,6 +254,11 @@
     - A custom value object takes two fields C(key) and C(value).
     - Incorrect key and values will be ignored.
     version_added: '2.3'
+  bootorder:
```

Comment:
rename this parameter to 'boot_order' to match another PR - https://github.com/ansible/ansible/pull/40609/files#diff-edefcb875c0dd6b5deefd1c19534792dR41